CDTDatastore/HTTP/CDTHTTPInterceptorContext.m

```diff
@@ -19,16 +19,18 @@
 
 @implementation CDTHTTPInterceptorContext
 
+NSMutableDictionary<NSString*, NSObject*> *_state;
```

Comment:
```objc
@property (class) NSString *someString
```

See [wwdc video](https://developer.apple.com/videos/play/wwdc2016/405/) about five minutes in.You'll need a newer LLVM compiler to be able to use it though, so it might be best to stick it as a ticket until Xcode 8 is final around september/october time.